### PR TITLE
Added new theme

### PR DESCRIPTION
**Pull request by divyakelaskar**
### Theme Name : Prussian
ss

### CODE REVIEW

**Comment by anuraghazra** [comment]
Hi @divyakelaskar thanks for the theme contribution, this looks awesome :)

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .vercel
 .env
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vercel
```
Comment (anuraghazra) [inline .gitignore]: Can you please revert these changes? .vercel is already in there

**Comment by anuraghazra** [comment]
LGTM! thanks @divyakelaskar

File: .gitignore (hunk 1 above)
Comment (divyakelaskar) [reply .gitignore]: :heavy_check_mark: Done !

**Comment by nombrekeff** [comment]
When adding a theme we should try adding it to the [`themes/README.md`](https://github.com/anuraghazra/github-readme-stats/edit/master/themes/README.md) file as well! So people can preview it :)

I can add this one myself when it's deployed.

**Comment by divyakelaskar** [comment]
Wasn't sure of adding it in README before deploying !

**Comment by nombrekeff** [comment]
Yes, I didn't realize it was not yet deployed. @anuraghazra Let us know when this theme has been deployed and I or @divyakelaskar will add the theme to the [preview](https://github.com/anuraghazra/github-readme-stats/edit/master/themes/README.md).

**Comment by anuraghazra** [comment]
@nombrekeff should be deployed now.

**Comment by divyakelaskar** [comment]
Thanks @anuraghazra !

**Comment by nombrekeff** [comment]
@divyakelaskar Do you want to add it yourself? Or should I add it?

**Comment by anuraghazra** [comment]
Do it! :) @nombrekeff

**Comment by divyakelaskar** [comment]
No problems. Go for it @nombrekeff 👍

**Comment by anuraghazra** [comment]
Next time probably we can ask the contributor to add the themes to theme readme :D that way the change is collocated

**Comment by divyakelaskar** [comment]
Sounds great !

**Comment by nombrekeff** [comment]
Yeah great suggestion, I was thinking of creating an ADD_YOUR_THEME guide, and explain all the steps to add it.

**Comment by anuraghazra** [comment]
> Yeah great suggestion, I was thinking of creating an ADD_YOUR_THEME guide, and explain all the steps to add it.

Just add it on the main theme readme :D

**Comment by nombrekeff** [comment]
Ohh definitely, I will do that. And we can link to it from the "Add Your Thems" links

**Comment by nombrekeff** [comment]
> > Yeah great suggestion, I was thinking of creating an ADD_YOUR_THEME guide, and explain all the steps to add it.
> 
> Just add it on the main theme readme :D

I will do that later though, I'm going now!
